### PR TITLE
Quickfix - timezones borking up import/export commands

### PR DESCRIPTION
**Pull request by alexiscatnip**
-well it will work even if your PC is set to travis-land now.
-Also, now able to output error message when 'timeslot overlap' or 'invalid timeslot' 

if i have time i will work on the tests tomorrow, but unlikely.

### CODE REVIEW

**Comment by alexiscatnip** [comment]
OMG TRAVIS YAS

**Comment by bennchong** [comment]
> OMG TRAVIS YAS

Achievement completed

**Comment by alexiscatnip** [comment]
> > OMG TRAVIS YAS
> 
> Achievement completed

*slow clap for meee*

File: src/main/java/seedu/address/logic/commands/ImportCommand.java [hunk 1]
```diff
@@ -61,28 +61,30 @@ public CommandResult execute(Model model, CommandHistory history) throws Command
         Person personToEdit = model.getUser();
         requireNonNull(personToEdit);
 
-        Optional<TimeTable> optionalTimeTable;
         TimeTable timeTable;
 
         try {
-            optionalTimeTable = IcsUtil.getInstance().readTimeTableFromFile(filePath);
+            //TODO: remove hardcoding of timezone in import and export command once TimeSlots are in UTC.
```
Comment (bennchong) [inline src/main/java/seedu/address/logic/commands/ImportCommand.java]: Is this todo finished?

**Review by bennchong — APPROVED** [review]

**Pull request merged**